index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/projects.css">
+  <link rel="stylesheet" type="text/css" href="./css/about.css">
+  <link rel="stylesheet" type="text/css" href="./css/form.css">
+  <title>Personal-Profolio</title>
+
+</head>
+
+<body>
+  <!-- Headline Section Starts Here -->
+
+  <header class="navigation">
+    <div class="logo">
+      <h2>My logo</h2>
+    </div>
```

Comment:
Instead of using a header tag `h2`, Let's consider making the Logo a link (`a` tag). 

This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 🤗🛠